ui/src/main/webapp/home/reducer.js

```diff
@@ -1,10 +1,12 @@
 import { combineReducers } from 'redux-immutable'
 import { Map } from 'immutable'
+import sub from 'redux-submarine'
 
-export const getSourceConfigs = (state) => state.getIn(['home', 'configs', 'sources'])
-export const getLdapConfigs = (state) => state.getIn(['home', 'configs', 'ldap'])
-export const getSubmittingPids = (state) => state.getIn(['home', 'submitting']).toJS()
-export const getConfigErrors = (state, servicePid) => state.getIn(['home', 'errors', servicePid], [])
+export const submarine = sub()
+export const getSourceConfigs = (state) => submarine(state).getIn(['configs', 'sources'])
```

Comment:
Here's where the home submarine is being set now in the root reducer - https://github.com/connexta/admin-console-beta/pull/40/files#diff-ea8e4875bf567254696c0561dd545292R29